openpype/hosts/maya/plugins/load/load_audio.py

```diff
@@ -61,6 +61,14 @@ def update(self, container, representation):
 
         path = get_representation_path(representation)
         cmds.setAttr("{}.filename".format(audio_node), path, type="string")
+
+        cmds.timeControl(
+            mel.eval("$tmpVar=$gPlayBackSlider"),
+            edit=True,
+            sound=audio_node,
+            displaySound=True
+        )
```

Comment:
Not too sure about doing this on update - I feel like it'd just be interfering with the scene state without a real reason. In the past I had cases where I had multiple audio files loaded using the **Use Trax Sounds** option so it'd play multiple audio files instead of just the one.

![image](https://github.com/ynput/OpenPype/assets/2439881/977c69a6-c2be-4ffc-8657-6310dab5dd64)

Having this code here would mean it'd just be changing that setting everytime you update the audio file?

Reply:
Thought I could be lazy about this, but you always catch me. Yeah, we'll need to check the audio node is used before updating.

Reply:
We could just remove this right? If the user had already set it to the audio node (e.g. from the `load` itself and not changing it) then updating shouldn't remove it right since the audio node remains the same node?

Reply:
Changing the file path makes the audio node not active on the timeline.